lib/tasks/data_migrations/12_populate_token_decimals_on_coins.rake

```diff
@@ -8,6 +8,7 @@ namespace :data_migrations do
 
     batch_process(coins_to_update) do |coin|
       PopulateTokenDecimalsForCoinWorker.new.perform(coin)
+      sleep 0.5
```

Comment:
I'm guessing this is to stay under the Etherscan rate limit?

Reply:
Sorry I meant Ethplorer

Reply:
Yup